src/codacy_pylint_test.py

```diff
@@ -136,27 +131,9 @@ def test_python3_file(self):
         self.assertEqual(withConfigAndSources(config, sources), expected_result)
 
     def test_no_conf(self):
-        (config, sources) = python3_file()
-        expected_result = [
-            Result('E0102.py', 'Trailing newlines', 'C0305', 23),
-            Result('E0102.py', '''Module name "E0102" doesn't conform to snake_case naming style''', 'C0103', 1),
-            Result('E0102.py', 'Missing module docstring', 'C0111', 1),
-            Result('E0102.py', 'Missing class docstring', 'C0111', 4),
-            Result('E0102.py', 'Missing method docstring', 'C0111', 5),
-            Result('E0102.py', 'Method could be a function', 'R0201', 5),
-            Result('E0102.py', 'method already defined line 5', 'E0102', 9),
-            Result('E0102.py', 'Missing method docstring', 'C0111', 9),
-            Result('E0102.py', 'Method could be a function', 'R0201', 9),
-            Result('E0102.py', 'Too few public methods (1/2)', 'R0903', 4),
-            Result('E0102.py', 'class already defined line 4', 'E0102', 13),
-            Result('E0102.py', 'Missing class docstring', 'C0111', 13),
-            Result('E0102.py', 'Too few public methods (0/2)', 'R0903', 13),
-            Result('E0102.py', 'Missing function docstring', 'C0111', 17),
-            Result('E0102.py', 'function already defined line 17', 'E0102', 21),
-            Result('E0102.py', 'Missing function docstring', 'C0111', 21)
-        ]
+        (_, sources) = python3_file()
         result = withConfigAndSources(None, sources)
-        self.assertEqual(result, expected_result)
+        self.assertTrue(len(result) > 0)
```

Comment:
uhmmm, at least we can test the number of issues, we are in control of the files right?

Reply:
Yes, we can. I checked only `> 0` because the issues usually change on every Pylint update.

Reply:
Changed to check length as:
```python
self.assertEqual(len(result), 16)
```
👍